docs/server-side/v2/active-rooms/get-peers.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Get Peers
+nav: 3.9
+---
+
+You can use this API for get the active peers in the room. Peers can be filtered by specifying user_id or role in the query params
+
+<PostRequest title="https://api.100ms.live/v2/active-rooms/<room_id>/peers" />
+<Request id="req-0">
+
+```bash
+curl --location --request GET 'https://api.100ms.live/v2/active-rooms/<room_id>/peers' \
+--header 'Authorization: Bearer <management_token>
+```
+
+</Request>
+<ResponseBox id="res-0" status="200 OK">
+
+```json
+{
+    "peers": {
+        "dcd88eb0-83e6-486a-bcae-c3be5e0f2c20": {
+            "id": "dcd88eb0-83e6-486a-bcae-c3be5e0f2c20",
+            "name": "John",
+            "user_id": "da3136fd-622e-468f-96ca-09c8c9f50072",
```

Comment:
shall we use some intuitive values for user_id field here, maybe something like student1 & student2?

Reply:
sure, I'll put something like `student-637`, `student-7810`